Civi/API/Kernel.php

```diff
@@ -213,7 +213,8 @@ protected function validate($apiRequest) {
    *   The full description of the API request.
    * @throws Exception\NotImplementedException
    * @return array
-   *   Array(0 => ProviderInterface, 1 => array).
+   *   A tuple with the provider-object and a revised apiRequest.
```

Comment:
Will the provider-object always be an instance of ProviderInterface? If so it might be helpful to mention that here